Lib/test/test_range.py

```diff
@@ -91,6 +91,13 @@ def test_range(self):
         r = range(-sys.maxsize, sys.maxsize, 2)
         self.assertEqual(len(r), sys.maxsize)
 
+    def test_range_constructor_error_messages(self):
+        with self.assertRaises(TypeError, msg="range expected at least 1 argument, got 0"):
```

Comment:
If this code is intended to check whether the error message is correct,
I'd like to suggest to use `assertRaisesRegex`.

Reply:
Woops! Thanks for the catch!